### DISCUSSION TITLE
Why can't pages with actions be pre-rendered?

### DISCUSSION THREAD

**Discussion by hyunbinseo** [sveltejs/kit#10915] [Q&A] [answered]
I've encountered this issue.

> Error: Cannot prerender pages with actions

It seems like static blog pages with a news-letter subscription `<form>` cannot be pre-rendered. Why is this?

**Comment by teemingc** [comment, marked as answer]
> Pages with actions cannot be prerendered, because a server must be able to handle the action POST requests.

https://kit.svelte.dev/docs/page-options#prerender-when-not-to-prerender

**Reply by hyunbinseo** [reply]
I will mark it as an answer, but I am still curious.

Is it technically impossible to have a pre-rendered HTML page that can make a POST form submission?

Or is it just a SvelteKit limitation as-of-now.

**Reply by teemingc** [reply]
You can have a form on a HTML page with the post method but you can’t export SvelteKit actions in +page.server.js if you choose to prerender it too
